### PR TITLE
added avatar, badge, buttons components

### PR DESCRIPTION
**Pull request by skiran252**
feat: added new badge, avatar and buttons component
docs: added components usage examples to docs/index.html

### CODE REVIEW

**Review by nikhilmalviya-dev — APPROVED** [review]
code is  clean and readable

File: styles/components/avatar.css [hunk 1]
```diff
@@ -0,0 +1,33 @@
+.avatar {
+    width: 2rem;
+    height: 2rem;
+    border-radius: 50%;
+    margin-right: 0.5rem;
```
Comment (Prabhav-Sharma) [inline styles/components/avatar.css:5]: Try to use global variables for properties like margins, padding and border-radius in common cases

**Review by cjinc121 — APPROVED** [review]
code is clean , you can go ahead with merging

**Comment by Kavita539** [comment]
Hey, I went through your code. Code structure and naming convention is good. You can go ahead and merge.

**Review by Prabhav-Sharma — COMMENTED** [review]
It is highly recommended to raise separate PR for each component. It will become too much work to review.

File: styles/components/avatar.css (hunk 1 above)
Comment (skiran252) [reply styles/components/avatar.css:5]: ya i thought to do . thank you . i ll do that

Comment (Prabhav-Sharma) [reply styles/components/avatar.css:5]: Okay

**Pull request merged**